src/main/java/com/hubspot/jinjava/tree/TagNode.java

```diff
@@ -84,4 +86,17 @@ public Tag getTag() {
     return tag;
   }
 
+
+  private String reconstructImage() {
+    StringBuilder builder = new StringBuilder().append(master.getImage());
```

Comment:
I wonder if we could store a reference to the original template string and an index value of the start and end n the node object. I think this would solve the raw case and preserve all the whitespace.

Reply:
I'll try to get that working in a separate PR. If I can get it to work, I'll loop back and update this PR to use it.

Reply:
Actually, it looks like it might require a larger version bump because I will need to mess around with Tag / Node etc constructors. I'll go ahead with this for now and come back to that as I refine this.